setup.py

```diff
@@ -29,7 +29,15 @@ def read_version():
 
 packages = setuptools.find_packages(where="src", exclude=("test",))
 
-required_packages = ["numpy", "six", "psutil", "retrying==1.3.3", "scipy"]
+required_packages = [
+    "model-archiver",
+    "multi-model-server",
+    "numpy",
+    "six",
```

Comment:
just checking if this is this still needed.

Reply:
Will remove Python 2 support in a separate PR.